src/DomainReader.ts

```diff
@@ -25,6 +25,7 @@ import { RoleDefinitionResolverV2__factory } from "../ethers/factories/RoleDefin
 import { ResolverContractType } from "./types/ResolverContractType";
 import { ERROR_MESSAGES } from "./types/ErrorMessages";
 import { ENSRegistry } from "../ethers/ENSRegistry";
+import { EWC_CHAIN_ID } from ".";
```

Comment:
A bit weird that this `import` is separate from the `VOLTA_CHAIN_ID` import. Could it be put with them?